CorsixTH/Lua/dialogs/fullscreen/graphs.lua

```diff
@@ -442,17 +466,7 @@ function UIGraphs:close()
 end
 
 function UIGraphs:afterLoad(old, new)
-  if old < 179 then
-    local gfx = TheApp.gfx
-
-    self.background = gfx:loadRaw("Graph01V", 640, 480, "QData", "QData", "Graph01V.pal", true)
-    local palette = gfx:loadPalette("QData", "Graph01V.pal", true)
-    self.panel_sprites = gfx:loadSpriteTable("QData", "Graph02V", true, palette)
-    self.white_font = gfx:loadFontAndSpriteTable("QData", "Font01V", false, palette)
-    self.black_font = gfx:loadFontAndSpriteTable("QData", "Font00V", false, palette)
-  end
-  UIFullscreen.afterLoad(self, old, new)
-  if old < 117 then
```

Comment:
Why are we deleting this? Bumping the 117 to 218 is fine though.

Reply:
Other updates in afterload between 179 and 218 may rely on updates here.